modules/bot_assistant/constants/commands.py

```diff
@@ -8,6 +8,10 @@
     "edit-phone": contact_handlers.edit_phone,
     "phone": contact_handlers.get_contact_phone,
     "remove-phone": contact_handlers.remove_phone,
+    "add-address": contact_handlers.add_address,
+    "edit-address": contact_handlers.edit_address,
+    "address": contact_handlers.get_contact_address,
```

Comment:
@KostiantynMoskalenko Recently naming convention changed, we define shows as `show-<entity>` now. E.g. `show-birthday`, `show-email`. Please update you functionality accordingly, thanks